homeassistant/components/lcn/scene.py

```diff
@@ -63,7 +65,7 @@ def __init__(self, config, address_connection):
     async def async_added_to_hass(self):
         """Run when entity about to be added to hass."""
 
-    async def async_activate(self):
+    def activate(self, **kwargs: Any) -> None:
```

Comment:
The pypck library supports asyncio via asyncio transports and protocols. This should run in the event loop.

Reply:
```py
    def send_command(self, pck):
        self.loop.create_task(self.send_command_async(pck))
```

It is indeed deeper into the pypck library. Thanks @MartinHjelmare 👍 

<https://github.com/home-assistant/core/pull/34493>